packages/smooth_app/lib/pages/product/product_page.dart

```diff
@@ -79,92 +79,102 @@ class _ProductPageState extends State<ProductPage> {
     _product ??= widget.product;
     return Scaffold(
         appBar: AppBar(
-          title: Text(
-            _product.productName ?? appLocalizations.unknownProductName,
```

Comment:
Are you sure that the productName cant be null

Reply:
[Yes I am](https://github.com/openfoodfacts/openfoodfacts-dart/blob/master/lib/model/Product.dart#L28-L28):
```dart
String? productName;
```
Beyond the `String?`, I think I had that problem during my tests. Or I had a `Text(null)` crash days before, which made me rather sensitive about that.